tests/shared/FlowManagerSettingsGetTestDefs.js

```diff
@@ -0,0 +1,465 @@
+/*
+ * GPII Flow Manager Get/Put shared est Definitions
+ *
+ * Copyright 2018, 2019 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013)
+ * under grant agreement no. 289016.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    jqUnit = fluid.registerNamespace("jqUnit");
+
+// These are test definitions for use with the cloud based flow manager in both
+// development and production configurations. The definitions are for getting
+// settings.
+
+// Define extra requests used for testing GET /settings endpoint
+fluid.defaults("gpii.tests.cloud.oauth2.settingsGet.requests", {
+    gradeNames: ["fluid.component"],
+    components: {
+        accessTokenRequest_settings: {
+            type: "kettle.test.request.http",
+            options: {
+                headers: {
+                    "Content-Type": "application/x-www-form-urlencoded"
+                },
+                path: "/access_token",
+                method: "POST"
+            }
+        },
+        settingsRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/%gpiiKey/settings/%device",
+                termMap: {
+                    gpiiKey: "{testCaseHolder}.options.gpiiKey",
+                    device: {
+                        expander: {
+                            func: "gpii.test.cloudBased.computeDevice",
+                            args: [
+                                [
+                                    "org.gnome.desktop.a11y.magnifier",
+                                    "org.gnome.desktop.interface",
+                                    "org.alsa-project"
+                                ],
+                                "linux"
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+    }
+});
+
+// For successful workflows that request user settings from /settings endpoint
+// using access tokens granted by /access_token endpoint
+fluid.defaults("gpii.tests.cloud.oauth2.settingsGet.mainSequence", {
+    gradeNames: ["fluid.test.sequenceElement"],
+    sequence: [
+        { funcName: "fluid.log", args: ["main sequence hit"]},
```

Comment:
Could you remove these debugging lines. Thanks.

Reply:
These were introduced when I merged in @the-t-in-rtf's GPII-3619 branch.  I thought they nicely logged the different tests segments. I see that you subsequently removed them (GPII-3721), but I missed that when I merged in master.  I think they are useful log information, but I'll remove them if you think it's necessary.

Reply:
I'm fine either way then. We can keep them if you feel they are useful to ppl.

Reply:
Having thought about it over the weekend, I think they are useful, but they currently are ambiguous.  For example, "main sequence hit" does not say which sequence.  I'll try to improve them, e.g., "Flow manager get settings main sequence".

Also, I found that the `ErrorTests.js` also use the technique in several places, e.g. [the logout-without-error-sequence](https://github.com/GPII/universal/blob/master/tests/ErrorTests.js#L117).